client/state/comments/reducer.js

```diff
@@ -126,7 +127,24 @@ export function totalCommentsCount( state = {}, action ) {
 	return state;
 }
 
+/**
+ * Keeps track of if a comment list request is in flight
+ * @param {object} state  - global redux state
+ * @param {object} action - redux action
+ * @returns {boolean} new redux state
+ */
+export function isCommentListLoading( state = false, action ) {
+	switch ( action.type ) {
+		case COMMENTS_LIST_REQUEST:
+			return true;
+		case COMMENTS_RECEIVE: //we may need a more specific success/error action here
+			return false;
+	}
```

Comment:
@dmsnell I mostly want us to be unblocked for user testing this week, and I don't have a ton of uninterrupted time to think about a generic fix in the time frame. I propose that we use the old pattern for now and revisit this in a refactor soon, so this can be addressed appropriately.

This would involve dispatching `COMMENTS_LIST_SUCCESS` and `COMMENTS_LIST_ERROR` and storing an isRequesting boolean per comment list site request. 

cc @Copons @rodrigoi if folks had better ideas.